styles.css

```diff
@@ -0,0 +1,175 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
+*{
+    margin: 0;
+}
+
+:root{
+    --primary: white;
+    --secondary:black ;

```

Comment:
extra spacing

Reply:
removed all the extra spacing thanks for ctrl+d